### ISSUE TITLE
Grindr V4

### ISSUE THREAD

**Issue by RobbieTechie** [tomlandia/fuckr#201]
So Grindr is apparently releasing version 4, this should be interesting.  It has a "Do not disturb" option, you can mark who you've chatted with, you also have the option to simply log out.

 I believe while I was ripping through the app I also saw the option to backup your chats, Either I overlooked it in the app itself, or it is not actually available yet.  Soon this likely means that V3 will likely get shut down, also meaning that likely this will kill the current version of Fuckr.

**Comment by Kav28** [comment]
Recently did the upgrade - chats, profiles were backed up to a "database".
Profile count was 52K which, because of its size, must be the "local"
count. Might be storing them for quicker display but I'd guess that the
tradeoff would be that initial info would be stale.

**Comment by RobbieTechie** [comment]
While I was thinking that they may add the option to back up chats, I believe what the backup was, was for the upgrade.  I didn't do an upgrade, I did a clean install.  I have the feeling these people at grindr are to stupid to add much of what their customer base actually wants.

  I've also noticed more sites for data collection and ads, probably so they can choose another system to sell your HIV status and testing info to someone else.  They also added a Chinese site www(.)qq(.)com (grindr was bought out by a chinese game company earlier this year) which posts data to pingma(.)qq(.)com/mstat/report/?index=(some id number here)  if you visit this with a browser you get a gzip error, meaning that this is likely for data collection
